google-datacatalog-kafka-connector/src/google/datacatalog_connectors/kafka/scrape/avro_schema_parser.py

```diff
@@ -0,0 +1,68 @@
+from avro import schema
+from google.datacatalog_connectors.kafka.config.\
+    metadata_constants import MetadataConstants
+
+
+class AvroSchemaParser:
```

Comment:
Can you add the test class this file: `avro_schema_parser_test.py`?

Reply:
Tests for this class are written already, they are in schema_parser_test.py. I have renamed this class but forgot to rename the corresponding test class, that's why names are not the same. I'll refactor and push changes

Reply:
@mesmacosta now tests fail with an error `AttributeError: module 'google.cloud.datacatalog_v1beta1' has no attribute 'enums'".` Does it maybe have to do with the fact that connectors use the different version of Google Cloud API now? I am not sure, what's the most sustainable way to fix it?

Reply:
Please set the fixed version on the `setup.py` file:

the `google-datacatalog-connectors-commons` should be `google-datacatalog-connectors-commons>=0.5.2,<0.6.0`

and `google-datacatalog-connectors-commons-test` should be `google-datacatalog-connectors-commons-test>=0.5.1,<0.6.0`

Versions greater than 0.6.0 use the new version of datacatalog client library.

Reply:
for reference:
[setup.py](https://github.com/GoogleCloudPlatform/datacatalog-connectors-rdbms/blob/rdbms-0.8.2/google-datacatalog-rdbms-connector/setup.py)

Reply:
On a new PR we can upgrade the kafka connector to work with the [2.0.0]( https://pypi.org/project/google-cloud-datacatalog/) version of datacatalog client library.

Reply:
thanks, Marcelo! all done